hermes-console-vue/src/composables/topic/use-topic/useTopic.spec.ts

```diff
@@ -0,0 +1,203 @@
+import { afterEach, describe, expect } from 'vitest';
+import {
+  dummySubscription,
+  secondDummySubscription,
+} from '@/dummy/subscription';
+import {
+  dummyTopic,
+  dummyTopicMessagesPreview,
+  dummyTopicMetrics,
+  dummyTopicOwner,
+} from '@/dummy/topic';
+import {
+  fetchTopicErrorHandler,
+  fetchTopicMessagesPreviewErrorHandler,
+  fetchTopicMetricsErrorHandler,
+  fetchTopicOwnerErrorHandler,
+  fetchTopicSubscriptionDetailsErrorHandler,
+  fetchTopicSubscriptionsErrorHandler,
+  successfulTopicHandlers,
+} from '@/mocks/handlers';
+import { setupServer } from 'msw/node';
+import { useTopic } from '@/composables/topic/use-topic/useTopic';
+import type { UseTopicErrors } from '@/composables/types';
+
+describe('useTopic', () => {
+  const server = setupServer(...successfulTopicHandlers);
```

Comment:
We will have to refactor other existing composables tests to use this `msw` server to be consistent (in different PR obviously)